identity/test/controllers/ConsentsJourneyControllerTest.scala

```diff
@@ -0,0 +1,219 @@
+package controllers
+
+import actions.AuthenticatedActions
+import com.gu.identity.cookie.GuUCookieData
+import com.gu.identity.model.Consent.Supporter
+import com.gu.identity.model.{EmailNewsletters, _}
+import controllers.editprofile.EditProfileController
+import form._
+import idapiclient.{Auth, TrackingData, _}
+import model.{Countries, PhoneNumbers}
+import org.mockito.Mockito._
+import org.mockito.{ArgumentCaptor, Matchers => MockitoMatchers}
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.mockito.MockitoSugar
+import org.scalatest._
+import org.scalatestplus.play.ConfiguredServer
+import play.api.http.HttpConfiguration
+import play.api.mvc._
+import play.api.test.Helpers._
+import services._
+import test._
+
+import scala.concurrent.Future
+
+@DoNotDiscover class ConsentsJourneyControllerTest extends WordSpec with WithTestExecutionContext
+  with Matchers
+  with MockitoSugar
+  with OptionValues
+  with ScalaFutures
+  with WithTestApplicationContext
+  with WithTestCSRF
+  with ConfiguredServer {
+
+  trait ConsentsJourneyFixture {
+
+    val controllerComponent: ControllerComponents = play.api.test.Helpers.stubControllerComponents()
+    val idUrlBuilder = mock[IdentityUrlBuilder]
+    val api = mock[IdApiClient]
+    val idRequestParser = mock[IdRequestParser]
+    val authService = mock[AuthenticationService]
+    val idRequest = mock[IdentityRequest]
+    val trackingData = mock[TrackingData]
+    val returnUrlVerifier = mock[ReturnUrlVerifier]
+    val newsletterService = spy(new NewsletterService(api, idRequestParser, idUrlBuilder))
```

Comment:
does this need to be a spy? I couldn't see where it was used.

Reply:
I honestly have no idea, I copypasted the boilerplate from editprofile. I'll check if it's used!